### DISCUSSION TITLE
Choose software to install on OS Installation

### DISCUSSION THREAD

**Discussion by trymeouteh** [linuxmint/discussions#325] [Ideas]
Please add a setting to install the minimum amount of software when installing Linux Mint.

**Comment by Fantu** [comment]
If you want cinnamon DE on a minimal installation without needed of all mint specific softwares you can try Debian.
Out of default installation, using cinnamon-desktop-environment metapackage including recommends, you can do some different installations easy. I had invested some days of testing plus the help of advices from some users during the years for some easy/fast choice.
cinnamon-desktop-environment without recommends for example to start without many additional softwares. cinnamon-core is more minimal and also it without recommends (and also cinnamon) was tested to have as little installation as possible but with cinnamon with most things working "out of the box".
As Debian support other archs (out of x86) and some users (for example on arm devices) have small disks (in some cases integrated nand chip), so require small installation.

**Reply by marco4mp** [reply]
But does Debian update Cinnamon versions?

**Reply by Fantu** [reply]
I haven't migrated from experimental in the last few weeks because of big 64-bit time_t migration in progress or the migration to testing remained blocked (except cinnamon-translation) and it was a problem like what happened on Ubuntu which seems to me to have been blocked for a month now in proposed. For armhf and armel was also not possible built, seems like they're fixing it these days.
As soon as the 64-bit time_t migration is finished I will upload 6.0 to unstable.
